client/src/test/VirtualEventHome.test.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { fireEvent, render, screen } from '@testing-library/react'
+import { rest } from 'msw'
+import { setupServer } from 'msw/node'
+
+import VirtualEventHome from '../components/VirtualEventHome';
+
+const handlers = [
+    rest.post('http://localhost:5000/events/create', (req, res, ctx) => {
```

Comment:
Can we get app domain from an env var?

Reply:
nvm lol

Reply:
nit: Can we use a mock domain name so it is clear that we know this is a mock server hostname?